packages/launcher/src/main/index.js

```diff
@@ -240,4 +342,6 @@ ipcMain.on('init-window', event => {
 
 ipcMain.on('ready-window', event => {
   BrowserWindow.fromWebContents(event.sender).show()
+  const menu = Menu.buildFromTemplate(template)
```

Comment:
This event is triggered for every app window we open, so rather than setting the menu here, I think it should be done in the `createLauncherWindow()` function, that gets called once the app is ready.